src/views/networkpolicies/new/NADsSelector.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC, useMemo } from 'react';
+
+import { NetworkAttachmentDefinitionModelGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { Alert, AlertVariant, FormGroup } from '@patternfly/react-core';
+import Loading from '@utils/components/Loading/Loading';
+import SelectMultiTypeahead from '@utils/components/SelectMultiTypeahead/SelectMultiTypeahead';
+import { DEFAULT_NAMESPACE } from '@utils/constants';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import { NetworkPolicy } from '@utils/models';
+import { NetworkAttachmentDefinitionKind } from '@utils/resources/nads/types';
+import { getName, getNamespace } from '@utils/resources/shared';
+
+type NADsSelectorProps = {
+  namespace: string;
+  networkPolicy: NetworkPolicy;
+  onPolicyChange: (policy: NetworkPolicy) => void;
+};
+
+const NADsSelector: FC<NADsSelectorProps> = ({ namespace, networkPolicy, onPolicyChange }) => {
+  const { t } = useNetworkingTranslation();
+
+  const [nads, loaded, loadError] = useK8sWatchResource<NetworkAttachmentDefinitionKind[]>({
+    groupVersionKind: NetworkAttachmentDefinitionModelGroupVersionKind,
+    isList: true,
+    namespace,
+  });
+
+  const [nadsDefault, loadedDefaultNads, loadErrorDefaultNads] = useK8sWatchResource<
+    NetworkAttachmentDefinitionKind[]
+  >(
+    namespace !== DEFAULT_NAMESPACE
+      ? {
+          groupVersionKind: NetworkAttachmentDefinitionModelGroupVersionKind,
+          isList: true,
+          namespace: DEFAULT_NAMESPACE,
+        }
+      : null,
+  );
+
+  const nadsOptions = useMemo(() => {
+    const allNads = [...(nads || []), ...(nadsDefault || [])];
+
+    return allNads.map((nad) => ({
+      children: `${getNamespace(nad)}/${getName(nad)}`,
```

Comment:
can be as const here and not used twice `${getNamespace(nad)}/${getName(nad)}`

Reply:
if its always the same value we need both value and children?

Reply:
We can make some changes in the SelectMultiTypeahead to just use value as children when there is no children